assets/javascripts/discourse/widgets/code-review-commit-approved-notification-item.js.es6

```diff
@@ -3,16 +3,15 @@ import { DefaultNotificationItem } from "discourse/widgets/default-notification-
 import { replaceIcon } from "discourse-common/lib/icon-library";
 import { postUrl } from "discourse/lib/utilities";
 import { userPath } from "discourse/lib/url";
+import { i18n } from "discourse/lib/computed";
 
 replaceIcon("notification.code_review_commit_approved", "check");
 
 createWidgetFrom(
   DefaultNotificationItem,
   "code-review-commit-approved-notification-item",
   {
-    notificationTitle() {
-      return I18n.t("notifications.code_review.commit_approved.title");
-    },
+    notificationTitle: i18n("notifications.code_review.commit_approved.title"),
```

Comment:
The `notificationTitle` isn't a property. It's consumed [here](https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/widgets/default-notification-item.js.es6#L111).

Reply:
hmmm right

why the params though ?

```
const title = this.notificationTitle(notificationName, data);
```

clearly seems not used 🤔 will remove

Reply:
OH OK nevermind I see, it's from core